source/uwp/Renderer/lib/MediaHelpers.cpp

```diff
@@ -224,25 +274,17 @@ namespace AdaptiveCards::Rendering::Uwp::MediaHelpers
                 auto getResourceStreamOperation = resourceResolver.GetResourceStreamAsync(args);
```

Comment:
@paulcam206 since we allow for custom resolver for `mediaSource` - should we allow for custom resolver for `captionSource` then as well?

Reply:
Thanks for pointing that out! Added